velox/docs/functions/spark/math.rst

```diff
@@ -266,6 +266,17 @@ Mathematical Functions
 
     Returns the secant of ``x``.
 
+.. spark:function:: sign(x) -> double
+
+    Returns the signum function of ``x``. Supported type for ``x`` is DOUBLE.
+    It returns:
+    * 0.0 if the argument is 0.0,
```

Comment:
Is there any difference on +0.0 and -0.0 for Spark?

Reply:
No, both return 0.0